kernelci.org/content/en/docs/instances/local.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Local (Dev setup)"
+date: 2021-08-12T10:15:37Z
+draft: true
+description: "How to set up a local KernelCI instance"
+---
+
+This section describes how to set up a KernelCI instance which is suitable for development. It's not meant to mimic the production-like setup on your local machine, but rather give directions on how to setup a minimal set of KernelCI components to simplify the process of development and testing new features.
+
+## Minimal set of KernelCI components
+
+The minimal setup for KernelCI consists of:
+
+- kernelci-backend
+- storage
+- kernelci-core
+
+This is sufficient for building the kernel, running tests and pushing build and test information as well as build artifacts to the backend and storage respectively.
+
+> # Note
+> These instructions are not suitable and should not be used to setup a production environment. They purposely neglect certain aspects of performance and security for the sake of simplicity.
+
+## Prerequisites
+
+Currently, the best option is to create a separate Virtual Machine that will run `kernelci-backend` and `storage`
+
+It's assumed in this document that you're running a following setup:
+
+- Virtual Machine running Debian Buster accessible via IP and domain name e.g. _kci-vm_. The following requirements need to be met:
+
+   - The VM must be reachable with a domain name from the host used for installation.
+   - SSH access is provided with key authorization
+
+> You can use DNS to access the VM with its domain name or simply put an entry to the `/etc/hosts` file.
+
+- Host machine which will be used to connect to the VM
+   It needs to meet following requirements:
+    - Installed apps:
+      - _git_
+      - _ansible_
+      - _Python 2.7_
+      - _Python 3.6+_
+
+## Deploy KernelCI backend and storage
+
+> # Note
+> It is assumed in this section that your kernelci VM is available from your host with the hostname `kci-vm`
+
+- Make sure _ansible_ is installed on the host machine.
+- Clone `kernelci-backend-config` repository
+
+    ```bash
+    git clone git@github.com:kernelci/kernelci-backend-config.git
+    ```
+
+- Configure ansible to be used with your VM
```

Comment:
Maybe convert the main steps as level-3 headers with `### Configure ansible` and then drop one level of indentation in the bullet points list.